pkgs/development/libraries/libiio/default.nix

```
@@ -4,7 +4,7 @@
 , flex
 , bison
 , libxml2
-, python
+, pythonSupport ? !stdenv.hostPlatform.isStatic, python
```

```suggestion
, pythonSupport ? stdenv.hostPlatform.hasSharedLibraries, python
```

`isStatic` is a bit confusingly-named... it is true only for `pkgsStatic`, not for platforms which lack a dynamic linker in the first place.  I'm not sure if Python works on any of those, though.